src/editor/jsoneditor.css

```diff
@@ -0,0 +1,251 @@
+:root {
```

Comment:
How much of this is in use? Do we need it? If so, can we move it the respective components (we can also create an issue for the backlog).

Reply:
I will remove the parts what we don't need, but we should keep this nevertheless. We need it for `MasterDetailLayout` in `TreeRenderer`. I will create an issue for this